service-compiler/src/main/java/io/techery/janet/JanetHttpProcessor.java

```diff
@@ -75,11 +83,29 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
                 actionClasses.add(actionClass);
             }
         }
-        if (!actionClasses.isEmpty()) {
-            httpHelpersGenerator.generate(actionClasses);
+        httpHelpersGenerator.generate(actionClasses);
+        httpHelpersFactoryGenerator.generate(actionClasses);
+        //
+        return false;
+    }
+
+    private List<TypeElement> findOtherHelpersFactories() {
+        PackageElement packageElement = elementUtils.getPackageElement(HttpActionService.HELPERS_FACTORY_CLASS_PACKAGE);
+        if (packageElement == null) return Collections.emptyList();
+        //
+        List<TypeElement> factoryElements = new ArrayList<TypeElement>();
+        for (Element element : packageElement.getEnclosedElements()) {
+            if (element.getKind().isInterface()) continue;
+            for (TypeMirror typeMirror : ((TypeElement) element).getInterfaces()) {
+                TypeElement interfaceElement = elementUtils
```

Comment:
instead of that `elementUtils.getTypeElement(HttpActionService.ActionHelperFactory.class.getCanonicalName());`